src/components/appNavigation/TabsView.js

```diff
@@ -105,23 +107,59 @@ const inviteButtonStyles = showRewardsFlag ? defaultLeftButtonStyles.slice(1) :
 //   </View>
 // )
 
-const RewardButton = ({ onPress, style }) => (
-  <>
-    <TouchableOpacity testID="rewards_tab" onPress={onPress} style={style}>
-      <Icon name="rewards" size={36} color="white" />
-    </TouchableOpacity>
-    <Appbar.Content />
-  </>
-)
+const RewardButton = React.memo(({ onPress, style }) => {
+  const [, , , inviteState] = useInvited()
+  const [updatesCount, setUpdatesCount] = useState(0)
 
-const InviteButton = ({ onPress, style }) => (
-  <>
-    <TouchableOpacity onPress={onPress} style={style}>
-      <Icon name="invite2" size={36} color="white" testID="invite_tab" />
-    </TouchableOpacity>
-    <Appbar.Content />
-  </>
-)
+  useEffect(() => {
+    const updateIcon = async () => {
+      const lastState = (await AsyncStorage.getItem('GD_lastInviteState')) || { pending: 0, approved: 0 }
+      const newPending = Math.max(inviteState.pending - lastState.pending, 0)
+      const newApproved = Math.max(inviteState.approved - lastState.approved, 0)
+      setUpdatesCount(newPending + newApproved)
+    }
+
+    updateIcon()
+  }, [inviteState])
+
+  return (
+    <>
+      <TouchableOpacity testID="rewards_tab" onPress={onPress} style={style}>
+        <Icon name="rewards" size={36} color="white">
+          {updatesCount > 0 && (
+            <View style={rewardStyles.notifications}>
+              <Text color={theme.colors.white} fontSize={10} fontWeight={'bold'}>
+                {updatesCount}
+              </Text>
+            </View>
+          )}
+        </Icon>
+      </TouchableOpacity>
+      <Appbar.Content />
+    </>
+  )
+})
+
+const rewardStyles = {
+  notifications: {
+    width: 13.3,
+    height: 13.3,
+    backgroundColor: theme.colors.orange,
+    borderRadius: 7,
+    position: 'absolute',
+    top: '-10%',
+    left: '70%',
+  },
+}
+
+// const InviteButton = ({ onPress, style }) => (
```

Comment:
i think we could remove all the stuff commented which was related to the eToro GoodMarket.
because now we're having **own** market on the Facebook and corresponding sticky button on the bottom.

@sirpy  your thoughts about this ?